build/single_file_libs/zstd-in.c

```diff
@@ -43,6 +43,8 @@
 #define ZSTD_MULTITHREAD
 #endif
 #define ZSTD_TRACE 0
+/* TODO: Can't amalgamate ASM function */
```

Comment:
Yeah, not "critical", but still an important follow-up

Reply:
Yeah, I'm not sure what to do about that. We may be able to get the amalgamation process to re-write the `.S` file as inline assembly.

I'm going to be opening an issue for follow ups, and I'll make sure to mention this.